routes/web.php

```diff
@@ -4,6 +4,10 @@
     'prefix' => config('attachments.routes.prefix'),
     'middleware' => config('attachments.routes.middleware')
 ], function () {
+    Route::get(config('attachments.routes.sharing.pattern'), 'Bnb\Laravel\Attachments\Http\Controllers\ShareController@download')
+        ->where('id', '[a-zA-Z0-9-]+')
```

Comment:
`token` is missing ? Keep `id` and `name` in case someone want's to use them in a custom config